types/webpack-env/index.d.ts

```diff
@@ -68,7 +68,7 @@ declare namespace __WebpackModuleApi {
         id: string;
         filename: string;
         loaded: boolean;
-        parent: NodeModule | null;
+        parent: NodeModule | null | undefined;
```

Comment:
I could have used
```suggestion
        parent?: NodeModule | null;
```
of course, but I was going for exact parity with the node types:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/1c2240f6cecbe84f5a5e257bdcef76709a57257b/types/node/ts3.1/globals.d.ts#L580